src/browser-installer/install.ts

```diff
@@ -0,0 +1,112 @@
+import _ from "lodash";
+
+/**
+ * @returns path to browser binary
+ */
+export const installBrowser = async (
+    browserName?: string,
+    browserVersion?: string,
+    { force = false, installWebDriver = false } = {},
+): Promise<string | null> => {
+    const unsupportedBrowserError = new Error(
+        [
+            `Couldn't install browser '${browserName}', as it is not supported`,
+            `Currently supported for installation browsers: 'chrome', 'firefox`,
+        ].join("\n"),
+    );
+
+    if (!browserName) {
+        throw unsupportedBrowserError;
+    }
+
+    if (!browserVersion) {
```

Comment:
> `browserVersion` is not required param

Currently it is with "--local".
browser version `latest` is hard to manage, because it will be different between multiple developers in one project, so it is an anti-pattern.

We could think about installing "latest" by default, but with `npx testplane` it will be treated as "latest installed version" (i want `npx testplane --local` to fully work in offline), so it could use `N` version, when `N + 1` is available

Reply:
I suggest to implement this in next PR